charts/secrets-store-csi-driver/templates/secrets-store.csi.x-k8s.io_secretproviderclasses.yaml

```diff
@@ -3,17 +3,13 @@
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  annotations:
-    controller-gen.kubebuilder.io/version: v0.2.4
   creationTimestamp: null
   name: secretproviderclasses.secrets-store.csi.x-k8s.io
 spec:
   group: secrets-store.csi.x-k8s.io
   names:
     kind: SecretProviderClass
-    listKind: SecretProviderClassList
```

Comment:
what version of controller-gen are you using? last time I checked v0.2.9 still generates these?

Reply:
It was an older version `v0.2.0`. Updated it to `v0.3.0`.